src/utils/proxy.ts

```diff
@@ -57,6 +57,12 @@ export async function proxyRequest(
     opts.headers,
   );
 
+  // DELETE method must always have an empty object in the body, and a "content-length" header.
+  if (method === "DELETE" && body === undefined) {
+    body = Buffer.from("{}");
```

Comment:
By HTTP spec, it is not disallowed. However conventionally, HTTP servers fail when `DELETE` omits the body. I'm not sure using a JavaScript HTTP object would be best solution.